.github/workflows/helm.yml

```
@@ -46,6 +46,8 @@ jobs:
 
       - name: Set up chart-testing
         uses: helm/chart-testing-action@v2.2.1
+        with:
+          version: v3.6.0
```

```suggestion
          version: ${{ env.helm-version }}
```
Let's use one version of helm across the workflow.

---

This is not the helm version, but the `chart-testing` version. The action still defaults to `v3.5.1`, but we are using the `--helm-extra-set-args` flag which was introduced in `ct v3.6.0`. I think we should just switch back to action's version only once it supports `ct v3.6.0` by default.

---

I have added a notice about removing the override once new version of `chart-testing-action` is released.